### ISSUE TITLE
Inconsistent links and link texts

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-brand-guidelines#240]
I found a couple of instances of link texts that are inconsistent (but not WCAG fails). Those are various links to WCAG:

* 'Web Content Accessibility Guidelines (WCAG 2.2) success criterion 1.4.3 Contrast (minimum) level AA' links to https://www.w3.org/TR/WCAG22/#contrast-minimum within [Colour > GOV.UK is a blue brand # Colour accessibility](https://govuk-brand-guidelines.netlify.app/colour/govuk-blue/#colour-accessibility).
* 'WCAG2.2 guidelines' links to https://www.w3.org/TR/WCAG22/#contrast-minimum (which is the same URL as above but with a different link text that misses that it's about colour contrast) within [Colour > GOV.UK is a blue brand # Incorrect colour usage](https://govuk-brand-guidelines.netlify.app/colour/govuk-blue/#incorrect-colour-usage)
* 'WCAG 2.2 Contrast (Minimum) Level AA' (another different link text with the same purpose as the above) links to https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html (which is inconsistently to the Understanding document, we should ideally only link to one of the two, unless this was intentional, in which case the link text should probably reflect the difference) within [Colour > Social # Companion colour set](https://govuk-brand-guidelines.netlify.app/colour/social/#companion-colour-set)
* 'WCAG 2.2: Use of Colour (Level A)' links to https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html (which is also the Understanding document rather than the direct success criterion) within [Colour > Social # Companion colour set](https://govuk-brand-guidelines.netlify.app/colour/social/#companion-colour-set)

This is out of a comment for #218.
